gammacat/collection.py

```diff
@@ -199,6 +199,45 @@ def validate_list_of_files(self):
         expected_files = expected_files_extra + expected_files_sed
         log_list_difference(actual, expected_files)
 
+class InputCollection:
+
+    def __init__ (self, config):
+        self.config = config
+        self.data = InputData.read()
+        self.info_files = InputInfoCollection.read()
```

Comment:
This line creates a list of all info files which are present in the input folder (step 1 in comment).